index.html

```diff
@@ -189,7 +190,7 @@ <h2>About Me</h2>
 				</div>
 			</div>
 
-			<div class="col-md-1"></div>
+			<div className="col-md-1"></div>

```

Comment:
In React, the attribute for defining CSS classes should be "className" instead of "class". Update it to "className="col-md-1"".